### PR TITLE
config: Make multiple claiming configurable.

### LINKED ISSUE #145: Make multiple person claiming configurable
The main issue is that new users will zulipbot claim an issue that has already been claimed, and then the person who is working on it has to go and say "hey, new-user, I'm working on this", which is sort of an awkward interaction. It's especially awkward I think when the old user has a completed PR, that's just waiting for review. E.g. sometimes they'll submit a PR, wait a week, and the first notification they get about it is someone else claiming their issue.

[Thread on chat.zulip.org](https://chat.zulip.org/#narrow/stream/105-zulipbot)

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
Fixes #145.

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -50,7 +50,8 @@ exports.issues = {
           labels: [],
           presence: false
         }
-      }
+      },
+      claimed: false
```
Comment (synicalsyntax) [inline config/default.js]: Can you change this to `multiple`?

File: src/commands/claim.js [hunk 2]
```diff
@@ -10,6 +11,15 @@ exports.run = async function(payload, commenter, args) {
     });
   }
 
+  if (payload.issue.assignees.length >= 1 && !claimed) {
```
Comment (synicalsyntax) [inline src/commands/claim.js]: `payload.issue.assignees.length` should be fine

**Comment by VibhorCodecianGupta** [comment]
Resolves #145

File: src/commands/claim.js (hunk 2 above)
Comment (VibhorCodecianGupta) [reply src/commands/claim.js]: Just a gentle reminder, made changes. Ready for review :)

File: src/templates/multipleClaimWarning.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Hello @{commenter}, it looks like someone is already assigned to this issue. We believe multiple assignments to the same issue causes confusion, and hence encourage you to go through other issues in this repository or other Zulip repositories and start working on them. Meanwhile, you can always return here and claim this issue in case it gets unassigned!
```
Comment (synicalsyntax) [inline src/templates/multipleClaimWarning.md]: @rishig Does this comment template look alright to you? Feel free to suggest changes if more improvement are needed

**Comment by synicalsyntax** [comment]
Merged! I fixed the commit message and added 44282d793362da5a77c01374a67dc52eb2b2536b to improve its functionality. Thanks so much @VibhorCodecianGupta :)

**Pull request closed without merging**